skops/card/_model_card.py

```diff
@@ -1315,3 +1315,36 @@ def render(self) -> str:
             sections inserted.
         """
         return "\n".join(self._generate_card())
+
+    def _iterate_key_section_content(
+        self,
+        data: dict[str, Section],
+        level: int = 0,
+    ):
+        for key, val in data.items():
+            if not getattr(val, "visible", True):
+                continue
```

Comment:
Let's add a test to cover this line. This visibility thing is not something we really advertise yet, but still good to test. To make a section invisible, select it and set it's visibility to `False`:

```
section = model_card.select(<section name>)
section.visible = False
```

Then this section should not appear in the toc.